designer/client/package.json

```diff
@@ -248,7 +248,7 @@
         "start:backend-remote": "npm run clean-translations && webpack serve",
         "start:backend-staging": "BACKEND_DOMAIN=https://staging.nussknacker.io npm run start:backend-remote",
         "start:backend-demo": "BACKEND_DOMAIN=https://demo.nussknacker.io npm run start:backend-remote",
-        "backend:docker": "docker run -i -p ${BE_PORT:-8080}:8080 -e CONFIG_FORCE_usageStatisticsReports_enabled=false -e SCHEMA_REGISTRY_URL=http://dummy:8888 -e KAFKA_ADDRESS=dummy:9092 -e OPENAPI_SERVICE_URL=http://dummy:5000 -e SQL_ENRICHER_URL=dummy:5432 -e CONFIG_FORCE_scenarioTypes_streaming__dev_deploymentConfig_type=stub -e CONFIG_FILE=/opt/nussknacker/conf/dev-application.conf -v `readlink -f ../../nussknacker-dist/src/universal/conf/dev-application.conf`:/opt/nussknacker/conf/dev-application.conf --pull ${DOCKER_PULL_OPTION:=always} -P touk/nussknacker:${NUSSKNACKER_VERSION:=staging-latest}",
+        "backend:docker": "docker run -i -p ${BE_PORT:-8080}:8080 -e CONFIG_FORCE_usageStatisticsReports_enabled=false -e SCHEMA_REGISTRY_URL=http://dummy:8888 -e KAFKA_ADDRESS=dummy:9092 -e OPENAPI_SERVICE_URL=http://dummy:5000 -e SQL_ENRICHER_URL=dummy:5432 -e CONFIG_FORCE_scenarioTypes_streaming__dev_deploymentConfig_type=stub -e CONFIG_FORCE_scenarioTypes_streaming__dev_modelConfig_components_sqlFile_disabled=true -e SQL_DATASOURCES_FILE_PATH=stub -e CONFIG_FILE=/opt/nussknacker/conf/dev-application.conf -v `readlink -f ../../nussknacker-dist/src/universal/conf/dev-application.conf`:/opt/nussknacker/conf/dev-application.conf --pull ${DOCKER_PULL_OPTION:=always} -P touk/nussknacker:${NUSSKNACKER_VERSION:=staging-latest}",
```

Comment:
I don't like passing a "stub" to `SQL_DATASOURCES_FILE_PATH`  besides disabling the component, but I don't see a simple alternative. At least it makes the cypress pipeline run and the example sql components are visible in dev category

Reply:
And theres no way to "clear" a value from config with this `CONFIG_FORCE`

Reply:
Regarding "stub", take a look at comment next to dev config.

Reply:
removed the stub :+1: